voluptuous/tests/tests.py

```diff
@@ -1,10 +1,13 @@
 import collections
 import copy
+import typing
 
+Enum: typing.Union[type, None]
```

Comment:
Oh, huh, I've never seen this syntax before, TIL.

Reply:
Usually you won't have to do this because the type can be inferred from context, but here it assumed it's always a `type` due to the import and then complains about the `= None` in the `except ModuleError`, so you'll have to be a bit more explicit :)